### PR TITLE
ENH: Include desc parameter in init_fitlins_wf 

### PR DESCRIPTION
**Pull request by AdinaWagner**
Related to PRs #92 and #94.
The 'desc' keyword gets included as a parameter in init_fitlins_wf.

### CODE REVIEW

**Comment by effigies** [comment]
Hi @AdinaWagner, thanks for these. I'm a little surprised that we were actually picking up files correctly...

Could you merge #92 and #94 into this branch? It's good for a pull request to create a new working state, if possible, and at least encapsulate all related changes. As it is, for instance, each of #92 and #94 will cause errors if merged by themselves, but I think with this one it would be a full fix.

**Comment by yarikoptic** [comment]
Just a bit of background - I specifically asked Adina to split into multiple PRs, so you could just choose the ones you like, since previous one #83 was left without merging until it got stale and still open

**Comment by yarikoptic** [comment]
But yeah, 92 and 94 should be cherry picked into this branch and closed ;-)

**Comment by effigies** [comment]
Yes, sorry about the delay on 83.

**Comment by AdinaWagner** [comment]
@effigies thx for the advice!
I might have followed @yarikoptic's orders to tightly ;-)

**Comment by AdinaWagner** [comment]
I'm closing #92 and #94 as their changes are now merged into this branch.

File: fitlins/cli/run.py [hunk 1]
```diff
@@ -91,6 +91,8 @@ def get_parser():
                         help='regex pattern to include files')
     g_bids.add_argument('--exclude', action='store', default=None,
                         help='regex pattern to exclude files')
+    g_bids.add_argument('--desc', action='store', default='preproc',
```
Comment (effigies) [inline fitlins/cli/run.py]: To be more consistent with `--participant-label`:

```suggestion
    g_bids.add_argument('--desc-label', action='store', default='preproc',
```

File: fitlins/cli/run.py [hunk 2]
```diff
@@ -91,6 +91,8 @@ def get_parser():
...
+                         help="desc field within BIDS-Derivatives files")
```
Comment (effigies) [inline fitlins/cli/run.py]:
```suggestion
                         help="use BOLD files with the provided description label")
```

File: fitlins/cli/run.py [hunk 3]
```diff
@@ -155,7 +157,9 @@ def run_fitlins(argv=None):
     work_dir = mkdtemp() if opts.work_dir is None else opts.work_dir
 
     fitlins_wf = init_fitlins_wf(
-        opts.bids_dir, derivatives, deriv_dir, opts.space, model=model,
+        opts.bids_dir, derivatives, deriv_dir, opts.space,
+        desc=opts.desc,
```
Comment (effigies) [inline fitlins/cli/run.py]: If you accept the above suggestion, this variable will change:

```suggestion
        desc=opts.desc_label,
```

Comment (AdinaWagner) [reply fitlins/cli/run.py]: Ha, I have learned something new about argparse today! I didn't know that an argument with a dash in the name can be parsed with an underscore. :-)

**Comment by AdinaWagner** [comment]
(hopefully I did everything correct with this "commit suggestion" functionality, let me know if not)

**Comment by effigies** [comment]
LGTM. Thanks!

**Pull request merged**